website/content/docs/upgrading/upgrade-to-1.18.x.mdx

```diff
@@ -0,0 +1,30 @@
+---
+layout: docs
+page_title: Upgrade to Vault 1.18.x - Guides
+description: |-
+  Deprecations, important or breaking changes, and remediation recommendations
+  for anyone upgrading to 1.18.x from Vault 1.17.x.
+---
+
+# Overview
+
+The Vault 1.18.x upgrade guide contains information on deprecations, important
+or breaking changes, and remediation recommendations for anyone upgrading from
+Vault 1.17. **Please read carefully**.
+
+## Important changes
+
+### Activity Log Changes
+
+The field `default_report_months` can no longer be configured or read. Any previously set values
+will be ignored by the system.
+
+
+Attempts to modify `default_report_months` through the
+[/sys/internal/counters/config](/vault/api-docs/system/internal-counters#update-the-client-count-configuration)
+endpoint, will result in the following warning from Vault:
+<CodeBlockConfig hideClipboard>
+```shell-session
+default_report_months is deprecated: defaulting to billing start time
```

Comment:
It might be more helpful to show an actual request/output, as a user might see it.

Not really a blocker, so if it's a pain I can approve once the other items are addressed.